ajax/libs/covervid/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "covervid",
+  "filename": "covervid.min.js",
+  "version": "1.0.0",
+  "description": "Make your HTML5 video behave like a background cover image with this lightweight Javascript plugin extension",
+  "author": "Stefan Erickson",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stefanerickson/covervid"
+  },
+  "keywords": [
+    "video",
+    "image",
+    "png"
+  ],
+  "npmName": "covervid",
```

Comment:
@pvnr0082t Could you please help me in removing the unwanted commits. I don't know how to do that thanks :)